<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_PumpControl</class>
 <widget class="QDialog" name="Dialog_PumpControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>278</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
background-color:#1d1d1d;
}

QMenuBar{
background-color:#1d1d1d;
padding:5px;
	font: 12pt &quot;MS Shell Dlg 2&quot;;
}

QMenuBar::item{
background-color:#1d1d1d;
color:#fff;
padding:5px;

}

QMenu{
color:#fff;
padding:0;
}

QMenu::item:selected{
color:#fff;
background-color:#00aba9;
}

QTableWidget{
background-color:#3d3d3d;
color:#fff;
  selection-background-color: #da532c;
border:solid;
border-width:3px;
border-color:#da532c;
}
QHeaderView::section{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(20, 158, 217, 255), stop:1 rgba(36, 158, 217, 255));
border:none;
border-top-style:solid;
border-width:1px;
border-top-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(20, 158, 217, 255), stop:1 rgba(36, 158, 217, 255));
color:#fff;

}
QHeaderView{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(20, 158, 217, 255), stop:1 rgba(36, 158, 217, 255));

border:none;
border-top-style:solid;
border-width:1px;
border-top-color:#149ED9;
color:#fff;
	font: 75 12pt &quot;Calibri&quot;;
}

QTableCornerButton::section{
border:none;
background-color:#149ED9;
}

QListWidget{
background-color:#3d3d3d;
color:#fff;
}

QMenu{
background-color:#3d3d3d;
}
QStatusBar{
background-color:#7e3878;
color:#fff;
}

QPushButton{
border-style:solid;
background-color:#3d3d3d;
color:#fff;
border-radius:7px;
}

QPushButton:disabled {
background-color: rgb(204, 0, 0);
}

QPushButton:hover{
color:#ccc;
	background-color: qlineargradient(spread:pad, x1:0.517, y1:0, x2:0.517, y2:1, stop:0 rgba(45, 45, 45, 255), stop:0.505682 rgba(45, 45, 45, 255), stop:1 rgba(29, 29, 29, 255));
	border-color:#2d89ef;
border-width:2px;
}

QPushButton:pressed{
background-color: qlineargradient(spread:pad, x1:0.517, y1:0, x2:0.517, y2:1, stop:0 rgba(29, 29, 29, 255), stop:0.505682 rgba(45, 45, 45, 255), stop:1 rgba(29, 29, 29, 255));
}


QTabWidget::tab{
background-color:#3d3d3d;
}

QLineEdit{
border-radius:0;
}

QProgressBar{
border-radius:0;
text-align:center;
color:#fff;
background-color:transparent;
border: 2px solid #e3a21a;
border-radius:7px;
	font: 75 12pt &quot;Open Sans&quot;;

}

QProgressBar::chunk{
background-color:#2d89ef;
width:20px;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="label_Electrolyte">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>170</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>170</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Electrolyte Type:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="lineEdit_Electrolyte">
         <property name="minimumSize">
          <size>
           <width>170</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>170</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_pH">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>170</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>170</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Electrolyte pH:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="lineEdit_pH">
         <property name="minimumSize">
          <size>
           <width>170</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>170</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_Conductivity">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>170</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>170</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Conductivity (mS/cm):</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="lineEdit_Conuctivity">
         <property name="minimumSize">
          <size>
           <width>170</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>170</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="1" rowspan="2">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_PumpRunning">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>140</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>18</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
           <underline>false</underline>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QMainWindow{
background-color:#1d1d1d;
}

QMenuBar{
background-color:#1d1d1d;
padding:5px;
	font: 12pt &quot;MS Shell Dlg 2&quot;;
}

QMenuBar::item{
background-color:#1d1d1d;
color:#fff;
padding:5px;

}

QMenu{
color:#fff;
padding:0;
}

QMenu::item:selected{
color:#fff;
background-color:#00aba9;
}

QTableWidget{
background-color:#3d3d3d;
color:#fff;
  selection-background-color: #da532c;
border:solid;
border-width:3px;
border-color:#da532c;
}
QHeaderView::section{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(20, 158, 217, 255), stop:1 rgba(36, 158, 217, 255));
border:none;
border-top-style:solid;
border-width:1px;
border-top-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(20, 158, 217, 255), stop:1 rgba(36, 158, 217, 255));
color:#fff;

}
QHeaderView{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(20, 158, 217, 255), stop:1 rgba(36, 158, 217, 255));

border:none;
border-top-style:solid;
border-width:1px;
border-top-color:#149ED9;
color:#fff;
	font: 75 12pt &quot;Calibri&quot;;
}

QTableCornerButton::section{
border:none;
background-color:#149ED9;
}

QListWidget{
background-color:#3d3d3d;
color:#fff;
}

QMenu{
background-color:#3d3d3d;
}
QStatusBar{
background-color:#7e3878;
color:#fff;
}

QPushButton{
border-style:solid;
background-color:#3d3d3d;
color:#fff;
border-radius:7px;
}

QPushButton:disabled {
background-color: rgb(204, 0, 0);
}

QPushButton:hover{
color:#ccc;
	background-color: qlineargradient(spread:pad, x1:0.517, y1:0, x2:0.517, y2:1, stop:0 rgba(45, 45, 45, 255), stop:0.505682 rgba(45, 45, 45, 255), stop:1 rgba(29, 29, 29, 255));
	border-color:#2d89ef;
border-width:2px;
}

QPushButton:pressed{
background-color: qlineargradient(spread:pad, x1:0.517, y1:0, x2:0.517, y2:1, stop:0 rgba(29, 29, 29, 255), stop:0.505682 rgba(45, 45, 45, 255), stop:1 rgba(29, 29, 29, 255));
}


QTabWidget::tab{
background-color:#3d3d3d;
}

QLineEdit{
border-radius:0;
}

QProgressBar{
border-radius:0;
text-align:center;
color:#fff;
background-color:transparent;
border: 2px solid #e3a21a;
border-radius:7px;
	font: 75 12pt &quot;Open Sans&quot;;

}

QProgressBar::chunk{
background-color:#2d89ef;
width:20px;
}</string>
         </property>
         <property name="text">
          <string>ON</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_OnTime">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>140</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>On Time:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_OnTime">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>140</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>13</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QWidget" name="widget_PumpOn" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>80</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>50</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QWidget" name="widget_PumpOn_2" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>80</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_flowRate">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="decimals">
          <number>0</number>
         </property>
         <property name="value">
          <double>0.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_Flowrate">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Flowrate Setpoint (lpm)</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_delayTime">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="decimals">
          <number>2</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_delayTim">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Delay Time (sec)</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QMainWindow{
background-color:#1d1d1d;
}

QMenuBar{
background-color:#1d1d1d;
padding:5px;
	font: 12pt &quot;MS Shell Dlg 2&quot;;
}

QMenuBar::item{
background-color:#1d1d1d;
color:#fff;
padding:5px;

}

QMenu{
color:#fff;
padding:0;
}

QMenu::item:selected{
color:#fff;
background-color:#00aba9;
}

QTableWidget{
background-color:#3d3d3d;
color:#fff;
  selection-background-color: #da532c;
border:solid;
border-width:3px;
border-color:#da532c;
}
QHeaderView::section{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(20, 158, 217, 255), stop:1 rgba(36, 158, 217, 255));
border:none;
border-top-style:solid;
border-width:1px;
border-top-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(20, 158, 217, 255), stop:1 rgba(36, 158, 217, 255));
color:#fff;

}
QHeaderView{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(20, 158, 217, 255), stop:1 rgba(36, 158, 217, 255));

border:none;
border-top-style:solid;
border-width:1px;
border-top-color:#149ED9;
color:#fff;
	font: 75 12pt &quot;Calibri&quot;;
}

QTableCornerButton::section{
border:none;
background-color:#149ED9;
}

QListWidget{
background-color:#3d3d3d;
color:#fff;
}

QMenu{
background-color:#3d3d3d;
}
QStatusBar{
background-color:#7e3878;
color:#fff;
}

QPushButton{
border-style:solid;
background-color:#3d3d3d;
color:#fff;
border-radius:7px;
}

QPushButton:disabled {
background-color: rgb(204, 0, 0);
}

QPushButton:hover{
color:#ccc;
	background-color: qlineargradient(spread:pad, x1:0.517, y1:0, x2:0.517, y2:1, stop:0 rgba(45, 45, 45, 255), stop:0.505682 rgba(45, 45, 45, 255), stop:1 rgba(29, 29, 29, 255));
	border-color:#2d89ef;
border-width:2px;
}

QPushButton:pressed{
background-color: qlineargradient(spread:pad, x1:0.517, y1:0, x2:0.517, y2:1, stop:0 rgba(29, 29, 29, 255), stop:0.505682 rgba(45, 45, 45, 255), stop:1 rgba(29, 29, 29, 255));
}


QTabWidget::tab{
background-color:#3d3d3d;
}

QLineEdit{
border-radius:0;
}

QProgressBar{
border-radius:0;
text-align:center;
color:#fff;
background-color:transparent;
border: 2px solid #e3a21a;
border-radius:7px;
	font: 75 12pt &quot;Open Sans&quot;;

}

QProgressBar::chunk{
background-color:#2d89ef;
width:20px;
}</string>
     </property>
     <property name="text">
      <string>DONE</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
